src/GTP.cpp

```diff
@@ -265,15 +265,16 @@ bool GTP::execute(GameState & game, std::string xinput) {
 
         if (!cmdstream.fail()) {
             if (tmp != 19) {
-                gtp_fail_printf(id, "unacceptable size");
+                gtp_fail_printf(id, 
```

Comment:
This violates the GTP protocol, see: 
https://www.lysator.liu.se/~gunnar/gtp/gtp2-spec-draft2/gtp2-spec.html#sec:list-of-all-commands

fails | Syntax error. If the engine cannot handle the new                 size, fails with the error message "unacceptable size".

It's the job of the GTP interface to translate these error messages to something more user friendly.

Using newlines in the responses seems to me like you risk breaking some GTP interfaces that don't expect this. I would avoid it if possible.